src/Pages/LandingPage/LandingPage.js

```diff
@@ -1,12 +1,15 @@
-import { Link } from "react-router-dom";
+import {Link} from "react-router-dom";
+import {SpotifyAuth} from "Components/index";
```

Comment:
Could you set your editor to leave spaces in object literals, imports and before the slash in components (see line 8 here)? Otherwise it's going to keep flipping back and forth whenever someone with a different editor edits it.

I may try to set-up something like Prettier to keep things consistent, but maybe that's a bit overkill 😅

Reply:
Sure, I'll check if I find the setting!

Reply:
I can't really find the setting for now but I'll take a deeper look into it later.